AdminWebsite/AdminWebsite/Controllers/HealthCheckController.cs

```diff
@@ -88,7 +93,22 @@ public async Task<IActionResult> Health()
                 }
             }
 
-            if (!response.UserApiHealth.Successful || !response.BookingsApiHealth.Successful || !response.VideoApiHealth.Successful)
+            try
+            {
+                await _notificationApiClient.CheckServiceHealthAuthAsync();
+            }
+            catch (Exception ex)
+            {
+                if (!(ex is NotificationApiException))
+                {
+                    response.NotificationApiClientHealth.Successful = false;
+                    response.NotificationApiClientHealth.ErrorMessage = ex.Message;
+                    response.NotificationApiClientHealth.Data = ex.Data;
+                }
+            }
```

Comment:
swallowing exception here if not NotificationApiException

Reply:
could maybe do something like 
`catch (Exception e) when (!(eis NotificationApiException))`
im not sure if the new logging / exception middleware is in adminweb yet but if it is it would automatically handle the exception logging and return internal server error if not handled in controller